server/resp/src/main/java/org/infinispan/server/resp/serialization/bytebuf/ByteBufThrowableSerializer.java

```diff
@@ -12,11 +14,14 @@
  * @author José Bolina
  */
 final class ByteBufThrowableSerializer implements ResponseSerializer<Throwable, ByteBufPool> {
+   private static final Log log = LogFactory.getLog(ByteBufThrowableSerializer.class);
    static final ByteBufThrowableSerializer INSTANCE = new ByteBufThrowableSerializer();
    private static final String DEFAULT_ERROR_MESSAGE = "failed handling command";
 
    @Override
    public void accept(Throwable throwable, ByteBufPool alloc) {
+      log.errorf(throwable, "Exception while handling a command");
```

Comment:
Did you mean to include this? If we do want to log as ERROR we need it added to the Log interface.

Reply:
I used it briefly to debug something. I can remove it.